### ISSUE TITLE
numpad backlight

### ISSUE THREAD

**Issue by ransur0t** [farfaaa/asus_zenbook_UM3402YA#7]
Hi!  Foremost, thank you for posting this firmware for the UM3402YA.  I've got the sound system functional on a recent install of Fedora 39.
  
And thank you for also posting the link to https://github.com/mohamed-badaoui/asus-touchpad-numpad-driver/ ... :)

I know the issue I am inquiring about is not relevant to this repo, but I am wondering if you had to modify the config to get the backlight to work?  I followed your suggested edit and also added a line to the python script per this thread https://github.com/mohamed-badaoui/asus-touchpad-numpad-driver/issues/71 ... but it still does not light up!

Any insight you can provide is much appreciated.

**Comment by ldrahnik** [comment]
Hi, do not know why you did not create an issue in the [repo](https://github.com/asus-linux-drivers/asus-numberpad-driver). I will try to help you here. 

Be you, I would first start this script [test brightness](https://github.com/asus-linux-drivers/asus-numberpad-driver/blob/master/tests/test_brightness.py). If it does nothing. I would continue manually.

So I mean (**<device_id>** is i2c device which can be found by running the script from repository `$ bash install_device_check.sh`) and then fill discovered **<device_id>** to this command where you have to change only previous last number **<0x01>**:

```
$ i2ctransfer -f -y <device_id> w13@0x15 0x05 0x00 0x3d 0x03 0x06 0x00 0x07 0x00 0x0d 0x14 0x03 <0x01> 0xad
```

And just manually try these values bewlo as replacment for **<0x01>** but keep in mind first has to be numberpad activated and then you can try changing level of backlight by these values:

0x01 should activate backlight of numberpad
0x60 should activate backlight of numberpad

0x41-0x48 - should change brightness levels, also everything abbelow might change brightness level in special cases
    "0x2f",
    "0x2e",
    "0x2d",
    "0x2c",
    "0x2b",
    "0x2a",
    "0x29",
    "0x28",
    "0x27",
    "0x26",
    "0x25",
    "0x24",
    "0x23",
    "0x22",
    "0x21",
    "0x20"

0x00 should deactivate backlight of numberpad
0x61 should deactivate backlight of numberpad

**Comment by ransur0t** [comment]
Thanks for the detailed reply.  I did reply to a thread on this same issue in the repo for the touchpad.  Sorry to trouble you here, and I appreciate your time.

I have since resolved the backlight failure by manually triggering the driver script during a debug attempt.  It started working immediately and has persisted across reboots.  Unfortunately the calculator launch no longer works, I think due to a broken association that was changed when I tried another touchpad driver ([https://github.com/asus-linux-drivers/asus-numberpad-driver](url).

Unfortunately I have a larger issue -- the MediaTek 7922 802.11ax WiFi adapter has pronounced issues with reliable speeds, sometimes slowing to a trickle, and other times not activating after wake.

It is a known issue, so either I keep the laptop and replace it with an Intel AX210 or do a return and continue my search for a replacement for my aging XPS 13 (9360)!
